services.yaml

```diff
@@ -1383,6 +1386,7 @@ paths:
     get:
       tags:
         - Views
+        - View Alerts
```

Comment:
why multiple tags?

Reply:
because this call provides you info about alert preferences on this view

Reply:
i'd stick with one tag per API for now. otherwise it gets weird.